vsintegration/src/FSharp.Editor/LanguageService/LanguageService.fs

```diff
@@ -425,7 +425,6 @@ type internal FSharpLanguageService(package : FSharpPackage) =
     member private this.OnProjectAdded(projectId:ProjectId) = projectInfoManager.UpdateProjectInfoWithProjectId(projectId, "OnProjectAdded", invalidateConfig=true)
```

Comment:
I think the invalidconfig can go away now, it seems as if it is always passed in as true.

Reply:
Yup, we can do that in master though. It shouldn't be a problem.